polaris-react/src/styles/shared/_buttons.scss

```diff
@@ -188,8 +188,8 @@
     }
 
     &.pressed {
+      border: var(--p-border-width-1) solid var(--p-border-critical);
       background: var(--p-surface-critical-subdued);
-      box-shadow: var(--p-border-width-1) solid var(--p-border-critical);
```

Comment:
Based on the values here I think this is supposed to be a `border` property not `box-shadow` (you can't use `solid` in `box-shadow`).